tests/test_misc.py

```diff
@@ -80,6 +81,7 @@ class MyModel:
     )
 
 
+@pytest.mark.skipif(sys.platform == 'emscripten', reason='README.md is not mounted in wasm file system')
```

Comment:
Instead of mounting the test directory maybe we could mount the root directory and then this would just work?

Reply:
I've tried that and run into https://github.com/samuelcolvin/pydantic-core/pull/118#issuecomment-1166523103

Reply:
Oh I see. How do fixtures work I don't understand?

Reply:
we can just skip that test, either if the files doesn't exist, or on wasm. Or just remove it if it's causing problems.

Reply:
Yeah of course it's fine to skip. But in the long term, we want to minimize the number of changes that have to happen to test suites to test against Pyodide.

Reply:
again, ignore my comment - I see you've done what I suggest.

Reply:
this is all related to pytest-benchmark, once that's working these workarounds can be removed.